binrw_derive/src/binrw/codegen/read_options/struct.rs

```diff
@@ -200,7 +200,7 @@ impl<'field> AfterParseCallGenerator<'field> {
                 }
             }
         } else {
-            get_args_argument(self.field, args_var)
+            get_args_argument(self.field, args_var, true)
```

Comment:
I'm not sure this has to be `true`. If it's in `after_parse`, that should be the last usage?

Reply:
If it doesn’t break the tests, either the tests are broken or it’s fine 🙃